resources/public/api/conf/3.0/examples/lossClaims/CreateLC/create_request_self_employment_claim.json

```diff
@@ -0,0 +1,6 @@
+{
+  "businessId": "XGIS00000001318",
```

Comment:
Can you update the businessId to XBIS12356589871 as XP is for Self-Employment related ids